src/webots/control/WbController.cpp

```diff
@@ -284,7 +284,7 @@ void WbController::start() {
       QFile file(runtimeFilePath);
       if (file.open(QIODevice::WriteOnly)) {
         QTextStream stream(&file);
-        stream << endl;
+        stream << Qt::endl;
```

Comment:
This is likely to fail on mac OS.
```suggestion
        stream << '\n';
```